gemonlinedb/html/scripts/gemonlinedb.js

```diff
@@ -0,0 +1,34 @@
+function sendrequest( jsonurl )
+{
+    if (window.jQuery) {
+        // can use jQuery libraries rather than raw javascript
+        $.getJSON(jsonurl)
+            .done(function(data) {
+                })
+            .fail(function(data, textStatus, error) {
+                    console.error("sendrequest: getJSON failed, status: " + textStatus + ", error: "+error);
+                });
+    } else {
+        var xmlhttp;
+        if (window.XMLHttpRequest) {// code for IE7+, Firefox, Chrome, Opera, Safari
+            xmlhttp=new XMLHttpRequest();
+        } else {// code for IE6, IE5
```

Comment:
Is support for such old browsers mandatory?

Reply:
I don't remember if it has any impact, I can't test IE/Edge

Reply:
This polyfill should be fine, but I'm concerned about having to check the layout on IE6 or even IE8 (since their implementation of the CSS Box Model was so bad) and not being able to use modern HTML5 / ES5 / CSS3 features